src/utils.ts

```diff
@@ -647,6 +648,32 @@ function isValidSnapshot(snapshot: number | string, network: string) {
   );
 }
 
+export function isStarknetAddress(address: string): boolean {
+  if (!address) return false;
+
+  try {
+    validateAndParseAddress(address);
+    return true;
+  } catch (e: any) {
+    return false;
+  }
+}
+
+export function isEvmAddress(address: string): boolean {
+  return isAddress(address);
+}
+
+export function getFormattedAddress(
+  address: string,
+  format: string[] = ['evm', 'starknet']
+): string {
+  if (format.includes('evm') && isAddress(address)) return getAddress(address);
```

Comment:
to keep it the same as line below this line
```suggestion
  if (format.includes('evm') && isEvmAddress(address)) return getAddress(address);
```